hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AuthenticationCustomCredentialsMessageTask.java

```diff
@@ -68,12 +68,12 @@ protected String getClientType() {
             clientName = parameters.clientName;
         }
 
-        if (parameters.attributesExist) {
-            Map<String, String> map = new HashMap<String, String>();
-            for (Map.Entry<String, String> attribute : parameters.attributes) {
-                map.put(attribute.getKey(), attribute.getValue());
+        if (parameters.labelsExist) {
```

Comment:
We can replace the iteration with 
labels = Collections.unmodifiableSet(new HashSet<String>(parameters.labels));